sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/models/SearchDocumentsResult.java

```diff
@@ -46,7 +47,7 @@
      * query parameter was not specified or set to 'none'.
      */
     @JsonProperty(value = "@search.answers", access = JsonProperty.Access.WRITE_ONLY)
-    private Map<String, List<AnswerResult>> answers;
+    private List<AnswerResult> answers;
```

Comment:
Is this intended?

Reply:
Yes, the Swagger was slightly incorrect and the proper wire type was array.

https://github.com/Azure/azure-rest-api-specs/issues/13848